File: 05.1차원_배열/01.최소,최대.c
#include <stdio.h>

int main(){
	int n,input;
	int max=-1000000,min=1000000;
	scanf("%d",&n);
	for(int i=0;i<n;i++){
		scanf("%d",&input);
		if(input>max)
		max=input;
		if(input<min)
		min=input;
	}
	printf("%d %d",min,max);
	return 0;
}

File: BaekJoon_Python/05.1차원배열/01.최소,최대.py
#! usr/bin/env python3


cnt = int(input())
numbers = list(map(int, input().split()))
max = numbers[0]
min = numbers[0]

for i in numbers[1:]:
        if i > max:
                max = i
        elif i< min:
                min = i

print(min,max)

"""
cnt = int(input())
num = list(map(int, input().split()))
print(min(num), max(num))
"""
